doc-content/drools-docs/src/main/asciidoc/DecisionEngine/phreak-sequential-mode-con.adoc

```diff
@@ -0,0 +1,85 @@
+[id='phreak-sequential-mode-con_{context}']
+
+= Sequential mode in Phreak
+
+Sequential mode is a rule base configuration in the {DECISION_ENGINE}, supported by Phreak, that enables rules in a stateless KIE session to access the data modified by the `modify` or `update` statements of previous rules. As a result, rule execution is faster in sequential mode because the data does not change after the initial data set insertion. Rules can be influenced by the execution of other rules for a more intuitive and holistic rule evaluation. Sequential mode also disables inference in rule evaluation.
+
+Sequential mode applies to stateless KIE sessions only because stateful KIE sessions inherently use data from previously invoked KIE sessions. If you use a stateless KIE session but you do not want the execution of previous rules to influence subsequent rules, then ensure that sequential mode is disabled. Sequential mode is disabled by default in the {DECISION_ENGINE}.
+
```

Comment:
the part on rule base config (sect. 2.2) is probably fine as it is. I would suggest rewording the two paragraphs above as follows:

> Sequential mode is an expert configuration of Phreak. It enables a special mode of execution that assumes no rule in the knowledge base affects the contents of the working memory; in other words, it is expected that no `insert`, `modify` or `update` statements occurs in any rule. If this is true, then the engine is allowed to assume that no rules will have to be re-evaluated, leading to a "sequential" execution, breaking out of the usual cyclic procedure. As a result, rule execution is faster in sequential mode because the data does not change after the initial data set insertion. However, if changes to the working memory *do* occur in any rule, then sequential mode should *never* be enabled.

> Sequential mode makes usually sense only in stateless KIE sessions only because stateful KIE sessions inherently use data from previously invoked KIE sessions. If you use a stateless KIE session but you do not want the execution of previous rules to influence subsequent rules, then ensure that sequential mode is disabled. Sequential mode is disabled by default in the {DECISION_ENGINE}. 

awaiting feedback from @mariofusco for this: is it fine?

Reply:
Updated, per our convo.